executor/src/witgen/sequence_iterator.rs

```diff
@@ -255,7 +269,9 @@ impl ProcessingSequenceCache {
         K: Copy + Ord,
         T: FieldElement,
     {
-        self.cache.entry(left.into()).or_default();
+        self.cache
+            .entry(left.into())
```

Comment:
Can it happen that we already have a cache entry here? What happens if it is non "incomplete"?

Reply:
I think that should never happen, I changed the code to panic in that case.